popup.js

```diff
@@ -40,31 +37,46 @@ function getMessage(total_count){
   }
 }
 
-function getData(handle){
+function getData(handle) {
 
-  var reqUrl = "https://api.github.com/search/issues?q=author%3A"+handle+"+type%3Apr"+
-              "+created%3A2017-09-30T00:00:00-12:00..2017-10-31T23:59:59-12:00+is%3Apublic";
+  var reqUrl = "https://api.github.com/search/issues?q=author%3A" + handle + "+type%3Apr+created%3A2017-09-30T00:00:00-12:00..2017-10-31T23:59:59-12:00+is%3Ap" +
+      "ublic";
 
   var req = new XMLHttpRequest();
-  req.onreadystatechange = function(){
-  	var res = "";
-    if(this.readyState == 4 && this.status == 200){
+  req.onreadystatechange = function () {
+    var res = "";
+    if (this.readyState == 4 && this.status == 200) {
       var data = JSON.parse(req.responseText);
       console.log(data);
       // document.getElementById("resultHandle").innerHTML = handle;
-      res +="<div id='resultHandle'>"+handle+"</div>";
-      var count = (data['total_count']>4?"4":data['total_count'])+ "/4";
-      res +="<div id='prCompleteCount'>"+count+"</div>";
+      res += "<div id='resultHandle'>" + handle + "</div>";
+      var count = (data['total_count'] > 4
+        ? "4"
+        : data['total_count']) + "/4";
+      res += "<div id='prCompleteCount'>" + count + "</div>";
       var message = getMessage(data['total_count']);
-      res +="<div id='message'>"+message+"</div>";
-	  if (data['items'].length > 0) {
```

Comment:
This condition should be there but is missing from your code. It is to show "Pull requests" only if number of PRs is more than 0.

Reply:
I added it back in.